Automattic/woocommerce-payments
# feat: payment methods selector add currency notice

Fixes #

#### Changes proposed in this Pull Request









Adding the notice about enabled currencies in the payment methods selector modal.

![2021-07-16 13 13 27](https://user-images.githubusercontent.com/273592/125991306-6de8db03-44b0-4b13-90ee-57e8660dc749.gif)


#### Testing instructions





- Ensure you have "Enable UPE settings changes" enabled
- Ensure you have "Enable UPE checkout" enabled
- Ensure you have "Enable Customer multi-currency" enabled
- Ensure your store's currency is set to "USD" and you don't have multiple currencies enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments
- Click the "Add payment method" button
- Select one of the available payment methods
- If an EU payment method is selected, the notice will be displayed

If the "multi currency" flag is disabled, you'll get a notice.
But we'll launch at the same time, so no biggie.
![Screen Shot 2021-07-16 at 1 42 01 PM](https://user-images.githubusercontent.com/273592/125994362-517c4716-c483-4236-82e1-b756cab7d7a3.png)


-------------------
